markovs_household/data/read_probabilities.py

```diff
@@ -0,0 +1,46 @@
+import os
+import pandas as pd
+import numpy as np
+from markovs_household.config.config import Configuration
+
+
+def calculate_probabilities(df, type):
+    config = Configuration.parse_config(r"tests\input\config.yaml")
+    usage_probability_str = "usage_probability_" + type
+
+    probabilities_df = pd.DataFrame(np.repeat(df.values, 4, axis=0))
+    probabilities_df.columns = df.columns
+
+    probabilities_df.insert(0, "spring_weekday", value=(probabilities_df["summer_weekday"] + probabilities_df["winter_weekday"]) / 2)
+    probabilities_df.insert(1, "spring_saturday", value=(probabilities_df["summer_saturday"] + probabilities_df["winter_saturday"]) / 2)
+    probabilities_df.insert(2, "spring_sunday", value=(probabilities_df["summer_sunday"] + probabilities_df["winter_sunday"]) / 2)
+    probabilities_df.insert(6, "autumn_weekday", value=(probabilities_df["summer_weekday"] + probabilities_df["winter_weekday"]) / 2)
+    probabilities_df.insert(7, "autumn_saturday", value=(probabilities_df["summer_saturday"] + probabilities_df["winter_saturday"]) / 2)
+    probabilities_df.insert(8, "autumn_sunday", value=(probabilities_df["summer_sunday"] + probabilities_df["winter_sunday"]) / 2)
+
+    probabilities_df = probabilities_df / 4 * getattr(config, usage_probability_str)
+
+    if not(os.path.isdir(r"tests\\input\\probabilities\\" + config.switch_on_probabilities_folder)):

```

Comment:
hier würde ich in der config den gesamten Pfad angeben lassen sodass der Nutzer sich aussuchen kann wo die Ergebnisse abgelegt werden